workSection.js

```diff
@@ -0,0 +1,100 @@
+const target = document.querySelector('.work');
+
+const payload = [
+  {
+    modal: {
+      gridItem: 'grid-item1',
+      modHeader: 'Tonic',
+      modpara1: 'CANCOPY',
+      modli1: 'Back End Dev',
+      modli2: '2015',
+      modpara2: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem excepturi cupiditate placeat suscipit!',
+      modli3: 'html',
+      modli4: 'css',
+      modli5: 'javascript',
+      backGround: 'img/Snapshoot Portfolio.png',
+      buttonNumber: 'work-button-1',
+    },
```

Comment:
- [ ] Kindly populate the array with link to the source code and the live preview as they are currently missing.

Reply:
## resolved 
-thank you for noticing these errors , I fixed it